load-testing/readme.md

```diff
@@ -0,0 +1,49 @@
+# Load Testing with Artillery
```

Comment:
This tutorial provides basic guidance on how to use Artillery. You can start by running `yarn artillery run test.yml` if you've created a new file. 

This project doesn't have any specific features; it simply contains a few starter YML files.

At present, we're not utilizing Influx and Grafana. My intention was to leverage Grafana and Influx's time-series database to establish a local widget. This widget would read and compare results from various runs, utilizing visualizations as opposed to reading numbers from output files

When you run 'yarn local', it will utilize your local system, however, it will only connect to remote URLs. Its good measure from your physical location.

If you run 'yarn aws', it allows you to measure data from regions such as Europe or Asia, regardless of your physical location.

Reply:
I don't mean I need documentation here I mean we need documentation in wiki or in ReadMe for these things.

Reply:
documentation updated